### PR TITLE
Feature/[NP-19]

### PR DESCRIPTION
**Pull request by Dhadds**
The app is now functioning. I will create a README that will help but we will inevitably have to overcome some setup issues when you guys move this onto your machines.

### CODE REVIEW

**Review by tanminivan — CHANGES_REQUESTED** [review]
It looks like we're still getting a lot of cruft added to the repo (pyc files, etc). It looks like we missed populating our `.gitignore` with data from gitignore.io to ignore common types. I went ahead and created a task for that (NP-35) and started PR to get that merged.

This will be a little bit complex to work in those changes to this branch so that we remove those unnecessary files, but we can work through it together if you'd like. The general idea is once my PR is merged, you'll want to rebase off of `main`, however the commit must be edited so that the ignored files will actually be ignored.

**Comment by tanminivan** [comment]
I'd also like to see these two commits squashed into one. Let me know if you want to do a screenshare and I can go over squashing/rebasing, this would be a good example.

File: frontend/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Use an official node runtime as a parent image
+FROM node:14
+
+WORKDIR /app/
+
+# Install dependencies
+# COPY package.json yarn.lock /app/
+
+# RUN npm install
```
Comment (chase-c55) [inline frontend/Dockerfile:9]: What are these extra comments for? Are we missing code or are they unnecessary?

**Review by chase-c55 — CHANGES_REQUESTED** [review]
Are you creating the README in the repo to set this up or will that be in Notion?

**Comment by tanminivan** [comment]
> Are you creating the README in the repo to set this up or will that be in Notion?

I think you could go either way on this, but a README in the repo feels like the better option to me, just so you can more easily track changes. Less likely to get out of date.

File: frontend/Dockerfile (hunk 1 above)
Comment (Dhadds) [reply frontend/Dockerfile:9]: I do not know.

**Review by tanminivan — APPROVED** [review]

File: frontend/Dockerfile (hunk 1 above)
Comment (Dhadds) [reply frontend/Dockerfile]: I removed these lines.

**Review by Koy-Wilson — APPROVED** [review]

**Review by chase-c55 — APPROVED** [review]

**Pull request merged**